lib/prototyp/hash.go

```diff
@@ -124,20 +124,20 @@ func (h Hash) IsValidTxnHash() bool {
 	return true
 }
 
-func (h Hash) Bytes() []byte {
```

Comment:
I'm not able to check it right now as I'm on holidays, however this may make indexer and other services using this new version of library not backward compatible.

I wouldn't change that. It's risky.

Reply:
I agree. I prefer not to make this change.. is it required..?

Reply:
I propose let’s keep this PR as a draft and we can reconsider it in the future